### PR TITLE
Added description of POA and DPOS consensus

### PR DESCRIPTION
**Pull request by keleffew**

### CODE REVIEW

File: consensus.asciidoc [hunk 1]
```diff
@@ -24,9 +24,24 @@ There are many kinds of consensus algorithms, and many ways to assign rewards to
 
 * In BFT-style proof of stake, validators are randomly assigned the right to propose blocks, but agreeing on which block is canonical is done through a multi-round process where every validator sends a "vote" for some specific block during each round, and at the end of the process all (honest and online) validators permanently agree on whether or not any given block is part of the chain. Note that blocks may still be chained together; the key difference is that consensus on a block can come within one block, and does not depend on the length or size of the chain after it.
 
-=== DPOS
+==== POA
+
+Proof of Authority (POA) is a subset of POS consensus algorithms utilized by primarily by testnets and private or consortium networks.  In POA-based blockchains, transaction validity is ulitmtely determined by a set of approved on-chain accounts, referred to as 'authority nodes'.  The criteria for determining authority nodes are decided deterministically through an approach codified in the network's governance structure.
```
Comment (fjrojasgarcia) [inline consensus.asciidoc]: "ulitmtely" is this a typo?

File: consensus.asciidoc [hunk 2]
```diff
@@ -24,9 +24,24 @@ There are many kinds of consensus algorithms, and many ways to assign rewards to
...
+
+POA is widely considerded to be the fastest route to consensus, but relies on the assumption that the validating node has not been compromised. Non-validating actors can access and use the network just as they would a public ethereum network (by leveraging p2p transactions, contracts, accounts etc.)
```
Comment (fjrojasgarcia) [inline consensus.asciidoc]: :+1:

File: consensus.asciidoc [hunk 3]
```diff
@@ -24,9 +24,24 @@ There are many kinds of consensus algorithms, and many ways to assign rewards to
...
+
+POA consensus relies on the validators reputation and past performance.  The idea is that the validator node is staking its identity/reputation to mine.  An important aspect in private consortium networks is the link between on-chain addresses to known, real world identities. Thus, We can say that the validating nodes are staking their "identity" or "reputation" (rather than their economic holdings). This creates some level accountability for validators and is best suited for enterprise, private, or test networks.
```
Comment (fjrojasgarcia) [inline consensus.asciidoc]: :+1:

File: consensus.asciidoc [hunk 4]
```diff
@@ -24,9 +24,24 @@ There are many kinds of consensus algorithms, and many ways to assign rewards to
...
+
+POA is currently employed by the test network Kovan, the POA network, and can be configured easily in Parity for private consortiums networks.
```
Comment (fjrojasgarcia) [inline consensus.asciidoc:35]: :+1:

File: consensus.asciidoc (hunk 1 above)
Comment (keleffew) [reply consensus.asciidoc]: yes, will update

File: consensus.asciidoc [hunk 5]
```diff
@@ -24,9 +24,24 @@ There are many kinds of consensus algorithms, and many ways to assign rewards to
...
+
+==== DPOS
+
+Delegated Proof of Stake (DPOS) is a modified form of Proof of Stake where network participants vote to elect an array of delegates (also called witnesses) to validate and secure the blockchain. These delegates are somewhat similiar to authority nodes in POA, except their authority may be revoked by the voters.
+
+In DPOS consensus, like in POS, the weight of the vote is proportional to the amount of stake injected by the user.  This creates a scenerio where larger token holders have proportionally more voting power than smaller ones.  This makes sense from a game theoretical perspective, as those with the more economic 'skin-in-the-game' will naturally have a larger incentive to elect the most efficient delegate witnesses.
```
Comment (fjrojasgarcia) [inline consensus.asciidoc]: Here did you mean "**scenario**" rather than "**scenerio**"?

Comment (fjrojasgarcia) [reply consensus.asciidoc]: Here I would add something like ...
" ... there are ways to make a “bribe” that are quite plausible; for example, an exchange can offer interest rates for deposits (or, even more ambiguously, use the exchange’s own money to build a great interface and features), with the exchange operator using the large quantity of deposits to vote as they wish in a DPOS consensus.

Comment (keleffew) [reply consensus.asciidoc]: done!

**Comment by fjrojasgarcia** [comment]
Thanks for your first contribution @keleffew!

**Pull request merged**